package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "traitify-widgets",
-  "version": "1.12.2",
```

Comment:
I don't think we need to bump this again since we didn't push the last commit to the registry